model/jpa/src/main/resources/META-INF/jpa-changelog-2.2.0.xml

```diff
@@ -57,5 +57,13 @@
 
     </changeSet>
 
+    <changeSet author="keycloak" id="2.2.0.1">
+        <addColumn tableName="REALM">
+            <column name="SSO_MAX_LIFESPAN_REMEMBER_ME" type="INT" defaultValueNumeric="3600"/>
+        </addColumn>
+
+        <!-- intialize the SSO_MAX_LIFESPAN_REMEMBER_ME = SSO_MAX_LIFESPAN for all pre-existing realms -->
+        <sql>update REALM set SSO_MAX_LIFESPAN_REMEMBER_ME = SSO_MAX_LIFESPAN where SSO_MAX_LIFESPAN is not null</sql>
```

Comment:
Would be simpler to allow null and use ssoSessionMaxLifespan if not specified. Having SQL in these files worries me for compatibility reasons (in general, I think this particular SQL is fine though).